full-service/src/config.rs

```diff
@@ -89,6 +89,10 @@ pub struct APIConfig {
     /// T3 Server to connect to and the api key to use for authorization.
     #[clap(flatten)]
     pub t3_sync_config: T3Config,
+
+    /// Webhook configuration.
+    #[clap(long, env = "ACCOUNT_SYNC_WEBHOOK_URL")]
+    pub account_sync_webhook_url: Option<String>, // FIXME: URL
```

Comment:
☝️ New configuration option: provide the webhook URL on full-service command line or in env var

Reply:
```suggestion
    pub deposits_webhook_url: Option<String>, // FIXME: URL
```

Reply:
Done! And added some documentation for this new feature